CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contributing
+
+First of all, thank you for contributing to the Software Gardening Almanac! 🎉 💯 🪴
+We're absolutely grateful for the kindness of your effort applied to grow this project into the best that it can be.
+
+This document contains guidelines on how to most effectively contribute to this project.
+
+If you are stuck or would like clarifications, please feel free to ask any questions or for help.
```

Comment:
Do you have any specific mechanism in mind to ask these questions? GitHub issues, discussions, something else?

Reply:
Thanks, good point! I wasn't sure about this and hesitated about being too prescriptive about how one should ask for help. In my opinion, anyone offering contributions should feel welcome to ask for help where they are (within an issue, a PR, discussion, chat, etc). I'll spell that out more clearly with an addition.

Reply:
I added some content here to help with this. Resolving for now.